lib/data_update_scripts/20210405152916_invoke_update_reading_list_document_trigger_on_articles.rb

```diff
@@ -0,0 +1,11 @@
+module DataUpdateScripts
+  class InvokeUpdateReadingListDocumentTriggerOnArticles
+    def run
+      return unless ActiveRecord::Base.connection.column_exists?(:articles, :reading_list_document)
+
+      # by updating `reading_list_document` to `NULL`,
+      # we invoke the `update_reading_list_document` trigger, bypassing Rails callbacks
+      Article.in_batches.update_all(reading_list_document: nil)
```

Comment:
The great thing about triggers is that they don't need Rails machinery, by seting the column to `NULL` we invoke the trigger that creates the full text search document and saves the correct value in the same column

Reply:
BTW @mstruve, given your comment [here](https://github.com/forem/forem/pull/13263#issuecomment-815068713), should I exit early if this is DEV? 

In this case we bypass callbacks (which I think are the source of the operation being slow). Let me know what you think.

Reply:
I would suspect this operation will be fast enough. The callbacks and find_each iterating over each record one by one is time-consuming. Check the DUS's in blazer after the job runs to make sure Postgres didn't timeout the operation but otherwise I think it's fine.